templates/base.html

```diff
@@ -34,7 +35,19 @@
 
             <!-- Footer -->
             <div class="footer">
-                <p><a href="{% url 'privacy' %}">Privacy</a></p>
+            <ul class="footer-items">
```

Comment:
I think we should change the indenting here to make this a little bit easier to read.
```html
<div class="footer">
    <ul class="footer-items">
        <li><a href="...
        <li><a href="...
        ...
    </ul>
    <div id="socials">
        <a href="...
        <a href="...
        ...
    </div>
</div>
```